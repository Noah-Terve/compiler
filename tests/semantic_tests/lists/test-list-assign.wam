list @l int @r a = [];
list @l int @r b = [1, 2, 100];
list @l char @r c = [];
list @l char @r d = ['a', 'b', 'c'];

c = d;