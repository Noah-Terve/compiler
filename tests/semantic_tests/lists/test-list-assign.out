(list @l int @r : a = (list @l int @r : []));
(list @l int @r : b = (list @l int @r : [(int : 1), (int : 2), (int : 100)]));
(list @l char @r : c = (list @l char @r : []));
(list @l char @r : d = (list @l char @r : [(char : 97), (char : 98), (char : 99)]));
(list @l char @r : c = (list @l char @r : d));
